src/Avalonia.Themes.Fluent/MenuItem.xaml

```diff
@@ -50,7 +50,6 @@
 
   <Style Selector="MenuItem">
     <Setter Property="Background" Value="{DynamicResource MenuFlyoutItemBackground}" />
-    <Setter Property="BorderBrush" Value="{DynamicResource MenuFlyoutItemRevealBorderBrush}" />
```

Comment:
Here and below I removed some setters with brushes, that were removed from resources earlier, because Avalonia doesn't support Reveal.